hermes/README.md

```diff
@@ -0,0 +1,46 @@
+# relayer
+
+`relayer` is a app developed for [Ignite CLI](https://github.com/ignite/cli).
+
+The app adds `ignite relayer` commands that allow launching new Cosmos blockchains by interacting with the Ignite Chain to coordinate with validators.
+
+The app is integrated into Ignite CLI by default.
+
+## How to use
+
+- run both chain to be relayed.
```

Comment:
Maybe we could use an upper case letter as the first letter for each one of the items, for example:

```suggestion
- Run both chain to be relayed.
```